packages/terra-data-grid/tests/jest/FlowsheetDataGrid.test.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import FlowsheetDataGrid from '../../src/FlowsheetDataGrid';
+
+// Source data for tests
+const dataFile = {
+  cols: [
+    { id: 'Column-0', displayName: 'Vitals', isResizable: true },
+    { id: 'Column-1', displayName: 'March 16' },
+    { id: 'Column-2', displayName: 'March 17', isResizable: true },
+  ],
+  rows: [
+    {
+      id: '1',
+      cells: [
+        { content: 'Heart Rate Monitored (bpm)' },
+        { content: '' },
+        { content: '66' },
+      ],
+    },
+    {
+      id: '2',
+      cells: [
+        { content: 'Temperature Oral (degC)' },
+        { content: '36.7' },
+        { content: '36.9' },
+      ],
+    },
+    {
+      id: '3',
+      cells: [
+        { content: 'Cardiac Index (L/min/m2)' },
+        { content: '2.25' },
+        { content: '2.28' },
+      ],
+    },
+    {
+      id: '4',
+      cells: [
+        { content: 'Oxygen Flow Rate (L/min)' },
+        { content: '63' },
+        { content: '47' },
+      ],
+    },
+  ],
+};
+
+describe('FlowsheetDataGrid', () => {
+  it('renders the row header column as pinned and remaining columns as overflow and all columns as not resizable', () => {
+    const wrapper = shallow(
+      <FlowsheetDataGrid
+        id="test-terra-flowsheet-data-grid"
+        rowHeaderColumn={dataFile.cols[0]}
+        overflowColumns={dataFile.cols.slice(1)}
+        rows={dataFile.rows}
+        ariaLabel="Test Flowsheet Data Grid"
+      />,
+    );
+
+    const dataGrid = wrapper.find('InjectIntl(DataGrid)');
+    const pinnedColumns = dataGrid.prop('pinnedColumns');
+    expect(pinnedColumns).toEqual(
+      [{ ...dataFile.cols[0], isResizable: false }],
```

Comment:
This test isn't really clear to me.  We should directly test that the columns have isResizable set to false.

Reply:
I can break this out into what is actually in the object array rather than using spread